textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/AlchemyAPIexample/alchemyapi.py

```diff
@@ -0,0 +1,780 @@
+#!/usr/bin/env python
```

Comment:
Let's not put this third-party code in the folder with our own code.  I had a chat with @zuozhiw and he said the python library has to be under the same folder.

I suggest we do the following: under this folder, create a subfolder called "third-party", and move this `alchemyapi.py` there.  Modify own code accordingly.  

@fukoyui can you make the changes?